ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -437,13 +437,14 @@ pattern CardanoNodeToClientVersion9 =
     HardForkNodeToClientEnabled
       HardForkSpecificNodeToClientVersion2
       (  EraNodeToClientEnabled ByronNodeToClientVersion1
-      :* EraNodeToClientEnabled ShelleyNodeToClientVersion5
-      :* EraNodeToClientEnabled ShelleyNodeToClientVersion5
-      :* EraNodeToClientEnabled ShelleyNodeToClientVersion5
-      :* EraNodeToClientEnabled ShelleyNodeToClientVersion5
-      :* EraNodeToClientEnabled ShelleyNodeToClientVersion5
+      :* EraNodeToClientEnabled ShelleyNodeToClientVersion6
```

Comment:
My intuition is that we should not change `CardanoNodeToClientVersion9`, but rather we should create a new `CardanoNodeToClientVersion10`. Did you choose to mutate 9 for some reason (eg perhaps 9 isn't released yet, so we can still change it?)

Reply:
There is now a new `CardanoNodeToClientVersion10`.